src/main/java/com/google/flourbot/entity/action/SheetAppendAction.java

```diff
@@ -0,0 +1,36 @@
+package com.google.flourbot.entity.action;
+
+import java.util.Map;
+import com.google.flourbot.entity.action.Action;
+
+
+public final class SheetAppendAction extends Action {
+
+    private final ActionType actionType = ActionType.SHEET_APPEND;
+
+    private final String[] columnValue; 
+    private final String sheetAction;
+    private final String sheetUrl;
+
+    public SheetAppendAction(String[] columnValue, String sheetAction, String sheetUrl) {
+        this.columnValue = columnValue;
+        this.sheetAction = sheetAction;
+        this.sheetUrl = sheetUrl;
+    }
```

Comment:
Specific action class now extends the Action abstract class. Incorporate ActionType Enum to define the type.